odl/operator/oputils.py

```diff
@@ -45,13 +50,34 @@ def matrix_representation(op):
            [4, 5, 6],
            [7, 8, 9]])
 
-    Works with product spaces:
-
-    >>> prod_ft = odl.DiagonalOperator(op)
-    >>> matrix_representation(op)
-    array([[1, 2, 3],
-           [4, 5, 6],
-           [7, 8, 9]])
+    It also works with `ProductSpace`s and higher dimensional `TensorSpace`s.
+    In this case, the returned "matrix" will also be higher dimensional:
+
+    >>> space = odl.uniform_discr([0, 0], [2, 2], (2, 2))
+    >>> grad = odl.Gradient(space)
+    >>> tensor = odl.matrix_representation(grad)
+    >>> tensor.shape
+    (2L, 2L, 2L, 2L, 2L)
```

Comment:
I can't imagine this will pass tests

Reply:
darn it.

Reply:
Did we ever decide on how to handle this type of problems?

Reply:
I don't know. Did Numpy always use long ints for shape entries, irrespective of Python version? I guess so, because it's in the C code. Maybe we never had a doctest that looked at the shape of a Numpy array?
I think the only sensible approach here is to not print stuff that may differ between Py versions (same with strings).